docusaurus.config.ts

```diff
@@ -18,6 +19,15 @@ const localePath = currentLocale === defaultLocale ? '' : currentLocale;
 
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
+const cfPagesBranch = process.env.CF_PAGES_BRANCH;
+
+console.log('#### Deploying branch ####', cfPagesBranch);
```

Comment:
do we still needs this?

Reply:
Yes, I need to see the console output in prod env (after deployment)

Reply:
can we limit this to CF only?

Reply:
Offline discussed and we move on. This console log will be removed later